index.bs

```diff
@@ -454,7 +454,7 @@ the subscriptions that have been made.
 Checking change subscriptions:
 
 ```js
-   const subscriptions = await cookieStore.getChangeSubscriptions();
+   const subscriptions = await cookieStore.getSubscriptions();
```

Comment:
Shouldn't this be: `await self.registration.cookies.getSubscriptions()` ?

(Examples are inconsistent about using `self`)

Reply:
oops, thanks for the catch ><